src/App.js

```diff
@@ -1,5 +1,7 @@
 import React, { Component } from 'react';
 import './App.css';
+import FriendsList from './FriendsList.js'
```

Comment:
When import from a javascript file, you don't need the file extension! 
So you could `import FriendsList from './FriendsList';`